pkg/ensurance/cm/cpu_assignment.go

```diff
@@ -0,0 +1,203 @@
+/*
```

Comment:
suggest to vendor the file instead of copy if no change

Reply:
there are two reason for me to do that:
1.I want to use CCD assignment cpu in the future
2.the function is private in k8s